demos/finserv-and-insurance/simple-loan-marketplace/README.md

```diff
@@ -0,0 +1,308 @@
+
+1. Click `GraphQL Explorer` on either node
+1. List all `Loan` items
+    ```
+    query ListLoans {
+      list_LoanItems {
+        _LoanItems {
+          _id
+          _owner
+          borrowerCreditScore
+          loanIdentifier
+          numberOfUnits
+          originalInterestRate
+          originalLoanToValue
+          originalUnpaidPrincipalBalance
+          originationDate
+        }
+      }
+    }
+    ```
+
+1. List all `LoanPerformance` items
+    ```
+    query ListLoanPerformance {
+      list_LoanPerformanceItems {
+        _LoanPerformanceItems {
+          _id
+          _owner
+          currentInterestRate
+          currentUnpaidPrincipalBalance
+          lastPaidInstallmentDate
+          loanDelinquencyStatus
+          loanIdentifier
+          monthlyReportingPeriod
+          servicerId
+        }
+      }
+    }
+    ```
+1. Filter `LoanPerformance` items by `currentInterestRate`
+    ```
+    query ListLoanPerformance {
+      list_LoanPerformanceItems(filter: {currentInterestRate: {lt: 3.00}}) {
+        _LoanPerformanceItems {
+          _id
+          _owner
+          currentInterestRate
+          currentUnpaidPrincipalBalance
+          lastPaidInstallmentDate
+          loanDelinquencyStatus
+          loanIdentifier
+          monthlyReportingPeriod
+          servicerId
+        }
+      }
+    }    
+    ```
+
+## Step 4 - Add a LoanPerformance Record
+You can also add new data to the Uni using either the Entity Explorer or GraphQL Explorer.
+
+### Add Data using Entity Explorer
+You can add data using the Entity Explorer view from either node in the Uni.
+
+1. Click `Entity Explorer` on either node
+1. Click `LoanPerformance` on the left-most pane
+1. Click `+ Create LoanPerformance`
+1. Enter values for each field
+    1. `loanIdentifier` - `23456`
+    1. `monthlyReportingPeriod` - `05/01/2022`
+    1. `servicerId` - `54321`
+    1. `loanDelinquencyStatus` - `current`
+    1. `lastPaidInstallmentDate` - `04/01/2022`
+    1. `currentInterestRate` - `2.75`
+    1. `currentUnpaidPrincipalBalance` - `551000`
+1. Click `Save`
+1. Wait for the indicator in the bottom-right to provide confirmation
+1. View the new `LoanPerformance` entry in the table
+
+### Add Data using GraphQL Explorer
+You can add data using the GraphQL Explorer view from either node in the Uni.
+
+1. Click `GraphQL Explorer` on either node
+1. Add a `LoanPerformance` item using the mutation below
+```
+mutation AddLoanPerformance {
+  add_LoanPerformance_async(
+    input: {lastPaidInstallmentDate: "2022-04-01", loanIdentifier: "23456", monthlyReportingPeriod: "2022-05-01", servicerId: "54321", currentInterestRate: 2.75, currentUnpaidPrincipalBalance: 551000, loanDelinquencyStatus: current}
+  ) {
+    result {
+      _id
+      _owner
+      submissionTime
+      transactionId
+      version
+    }
+    error
+  }
+}
+
+```
+1. Confirm the `LoanPerformance` item is included in the list of all items
```

Comment:
Markdown allows you to lead all numbered lists with a `1.`  It helps in case you need to adjust numbering and don't want to re-number all elements of the numbered list.  For example, check this out:

```
1. First
1. Second
1. Third
```

1. First
1. Second
1. Third

Reply:
I believe this is an indentation issue with the code snippet.  Let me try again.